BE/src/main/java/kr/codesquad/sidedish/domain/Item.java

```diff
@@ -37,51 +39,11 @@ public Item(Long itemId, Long categoryId, String itemName, String description, L
         this.images = images;
     }
 
-    public Long getItemId() {
-        return itemId;
+    public void changeStock(Integer quantity) {
+        this.stock = this.getStock() - quantity;
```

Comment:
getter가 아닌 객체 필드로 사용해주세요~

```suggestion
    public void changeStock(Integer quantity) {
        this.stock = this.stock - quantity;
```